stable/mlrun/values.yaml

```diff
@@ -0,0 +1,75 @@
+replicas: 1
+
+image:
+  repository: mlrun/mlrun-api
+  tag: 0.4.3
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+
+rbac:
+  create: true
+
+serviceAccount:
+  create: true
+  # If not set and create is true, a name is generated using the fullname template
+  name:
```

Comment:
best to comment this out and condition on existence then empty string in helpers imo - seeing this is a bit ugly with no value `name: `

Reply:
auto generated by `helm3 create mlrun` and it's like that in almost every official helm chart, keeping as is

Reply:
ugh. ok to this one